### ISSUE TITLE
Review/Add more libraries

### ISSUE THREAD

**Issue by brc-dd** [ddahan/ui-libs#4]
Hi, there! The following libraries can also be added to the data 🙌

- [ ] https://github.com/chakra-ui/zag -- https://zagjs.com/
- [ ] https://github.com/chakra-ui/ark -- https://ark-ui.com/
- [ ] https://github.com/chakra-ui/chakra-ui-vue-next -- https://next.vue.chakra-ui.com/
---
- [ ] https://github.com/radix-vue/radix-vue -- https://radix-vue.com/
- [ ] https://github.com/oku-ui/primitives -- https://oku-ui.com/primitives
- [ ] https://github.com/agnosticui/agnosticui -- https://www.agnosticui.com/
---
- [ ] https://github.com/jd-solanki/anu -- https://anu-vue.netlify.app/
- [ ] https://github.com/uiv-lib/uiv -- https://uiv.wxsm.space/
- [ ] https://github.com/onu-ui/onu-ui -- https://onu.zyob.top/
---

**Comment by ddahan** [comment]
Hi there. Thanks for all the suggestions.
I don't know all libraries but I'll take a look for sure.

However, I'd like to point out that the purpose of this tool is not to be absolutely comprehensive by listing all UI libraries, but rather propose a curated list to users to help them make a choice. I'm afraid that if in the end there are too many libraries, it gets complicated for users to make this choice.

Among all the libraries you listed, do you think one should be absolutely present because it has something particular that others don't?  Thanks.

**Comment by brc-dd** [comment]
I guess we can have some kind of threshold like number of stars or downloads to avoid adding too much libraries.

> Among all the libraries you listed, do you think one should be absolutely present because it has something particular that others don't?

Yeah I can explain briefly about each. Feel free to choose from them.

- zag (primitives), ark (components built on top of zag), chakra UI (next) (components built on top of zag) -- these are built by the chakra team and are more accessible than any of the libraries that I know for vue. (like react-aria kind of a11y). Although chakra-ui for vue3 is still in beta.
- radix-vue and oku seem to have similar goals of porting radix to vue. These are primitives. radix-vue is used by shadcn-vue (already on the list).
- agnostic ui also have primitives. Although the focus is more on cross framework support rather than a11y or design.
- anu and onu are built on top of unocss. uiv is on top of bootstrap. These are pre-designed components.

**Comment by ddahan** [comment]
Thanks for the summary!
I updated the README to be more specific about which library to include or discar.
Would you mind reading [this section](https://github.com/ddahan/ui-libs#why-library-x-is-not-listed) and make a first filter based on what it is said?  
Thanks.

**Comment by brc-dd** [comment]
IG we can keep these:

- Ark UI
- Anu
- uiv

The others don't seem to meet the criteria

- Zag - low-level primitives not components
- Chakra - WIP
- Radix Vue - WIP
- Oku - WIP
- Agnostic UI - no new commit in last six months
- Onu UI - WIP

**Comment by ddahan** [comment]
Thanks for the first filter! I'll take a look to these 3 libraries.

**Comment by ddahan** [comment]
@kissu You're right. It seems that Radix-Vue is evolving particularly quickly, as there are much more components that last time I checked. I'm considering adding it.

**Comment by ZainW** [comment]
would highly recommend adding oku primitives and ark-ui to the list

while there are a lot of headless libraries popping up right now these in particular are extremely important to the ecosystem, these are the kind of libraries that bridge the gap between Vue ecosystem and React ecosystem.

**Comment by bobbyrahmanda13** [comment]
What do you think about the [Vexip-ui](https://github.com/vexip-ui/vexip-ui/) which is already in the nuxt.com module

**Comment by ddahan** [comment]
Thanks @bobbyrahmanda13 I have added vexip-ui to the review list, in addition to inkline suggested by @alexgrozav

But as you can all see, the problem becomes more visible as more libraries are waiting for review: adding these 7 libraries to the list won't help users at all, as the choice will become harder for them. This app must not be a full directory of all libraries, but a curated list of advised one. That's how I designed it, and I don't want this to change.

I genuinely understand everyone wants to see its favorite library (or the one he has worked on) listed, but it will be counterproductive for end users.

I'm thinking of an arbitrary rule of:
- existing usage/popularity (XX stars and/or XX downloads) threshold
- except if the library has something very particular that worth mentioning it despite low popularity / newness (for example, Nuxt UI is not very popular yet but is the only library dedicated to Nuxt, which brings a great DX for Nuxt users, and is made by the Nuxt team itself).

If you have another idea that take care of users first, I'm open to discussion. Thanks.

**Comment by chenjiahan** [comment]
I propose **Vant**: _A lightweight, customizable Vue UI library for mobile web apps._

- repo: https://github.com/youzan/vant
- site: https://vant-ui.github.io/vant/#/en-US
- nuxt module: https://nuxt.com/modules/vant

**Comment by kissu** [comment]
> Thanks @bobbyrahmanda13 I have added vexip-ui to the review list, in addition to inkline suggested by @alexgrozav
> 
> But as you can all see, the problem becomes more visible as more libraries are waiting for review: adding these 7 libraries to the list won't help users at all, as the choice will become harder for them. This app must not be a full directory of all libraries, but a curated list of advised one. That's how I designed it, and I don't want this to change.
> 
> I genuinely understand everyone wants to see its favorite library (or the one he has worked on) listed, but it will be counterproductive for end users.
> 
> I'm thinking of an arbitrary rule of:
> 
> * existing usage/popularity (XX stars and/or XX downloads) threshold
> * except if the library has something very particular that worth mentioning it despite low popularity / newness (for example, Nuxt UI is not very popular yet but is the only library dedicated to Nuxt, which brings a great DX for Nuxt users, and is made by the Nuxt team itself).
> 
> If you have another idea that take care of users first, I'm open to discussion. Thanks.

Having an exhaustive list might help people have more visibility as of what is available tho. But I also understand your POV of keeping only some curated ones.

I think that having some notes per project (as for Nuxt UI) to give more context as of why it is nice, may be good. ✅ 
Otherwise, you can always put a ribbon 🎗 on some specific projects like "recommended by the maintainer"?👌🏻

**Comment by ddahan** [comment]
@kissu About the ibbon, good idea but the only problem is people will get crazy if their lib has no ribbon while they think it should has. As an example, Vuetify creator was angry to see nuxt UI in the first place (while this is absolutely not a ranking...). He tweeted about it.
By the way I don't know all the listed libraries well enough to be a judge, so I try to stick with very objective characteristics like.

**Comment by ddahan** [comment]
> Having an exhaustive list might help people have more visibility as of what is available tho

That's true too. Maybe (just maybe) a solution could be to list all the libraries, and have a default filter that only shows the "recommended ones". Or a more objective filters like "github stars count".
Again, there will be drama but...

**Comment by Baroshem** [comment]
Hey folks,

I would like to recommend to add Storefront UI. It is somehow unique because it targets E-Commerce applications so maybe it would be useful to add as other libs (from what I discovered) are more generic and project agnostic :)

**Comment by ddahan** [comment]
Hi there! I'm currently working on a v2 of ui-libs to include all decent libraries, but in a different way.
I close the issue because it will need different things, but be sure that if it make sense to include it, it will be.
Thanks!
